src/_ert/events.py

```diff
@@ -9,30 +9,59 @@
 
 from pydantic import BaseModel, Field, TypeAdapter
 
-_FORWARD_MODEL_START = Literal["com.equinor.ert.forward_model_job.start"]
-_FORWARD_MODEL_RUNNING = Literal["com.equinor.ert.forward_model_job.running"]
-_FORWARD_MODEL_SUCCESS = Literal["com.equinor.ert.forward_model_job.success"]
-_FORWARD_MODEL_FAILURE = Literal["com.equinor.ert.forward_model_job.failure"]
-_FORWARD_MODEL_CHECKSUM = Literal["com.equinor.ert.forward_model_job.checksum"]
 
-_REALIZATION_FAILURE = Literal["com.equinor.ert.realization.failure"]
-_REALIZATION_PENDING = Literal["com.equinor.ert.realization.pending"]
-_REALIZATION_RUNNING = Literal["com.equinor.ert.realization.running"]
-_REALIZATION_SUCCESS = Literal["com.equinor.ert.realization.success"]
-_REALIZATION_UNKNOWN = Literal["com.equinor.ert.realization.unknown"]
-_REALIZATION_WAITING = Literal["com.equinor.ert.realization.waiting"]
-_REALIZATION_TIMEOUT = Literal["com.equinor.ert.realization.timeout"]
-
-_ENSEMBLE_STARTED = Literal["com.equinor.ert.ensemble.started"]
-_ENSEMBLE_SUCCEEDED = Literal["com.equinor.ert.ensemble.succeeded"]
-_ENSEMBLE_CANCELLED = Literal["com.equinor.ert.ensemble.cancelled"]
-_ENSEMBLE_FAILED = Literal["com.equinor.ert.ensemble.failed"]
-
-_EE_SNAPSHOT = Literal["com.equinor.ert.ee.snapshot"]
-_EE_SNAPSHOT_UPDATE = Literal["com.equinor.ert.ee.snapshot_update"]
-_EE_TERMINATED = Literal["com.equinor.ert.ee.terminated"]
-_EE_USER_CANCEL = Literal["com.equinor.ert.ee.user_cancel"]
-_EE_USER_DONE = Literal["com.equinor.ert.ee.user_done"]
+class Id:
```

Comment:
We often import identifiers as Ids, so I think that would be a more suitable name.
```suggestion
class Ids:
```

Reply:
I'm not sure about this one. We do not use plural for other states like JobState and ObservationStatus for example.

Reply:
The class more or less replaces identifiers.py, so I think it should have the same naming convention. 
We could also change it to something like `EventIdentifier` instead.

Reply:
I do not think it should be plural unless that is something we use for other enums or classes filled with constants. It is already imported from event, so I think EventIdentifier would be a little repetitive, and also verbose :)

Reply:
Okay, looks good then 👌 !